### PR TITLE
Add feature variant for voice

### LINKED ISSUE #439: Make an extra artifact for voice
Voice is probably the least used feature, and we can remove the dependency pretty easily by using https://docs.gradle.org/current/userguide/feature_variants.html
Credit @lost-illusi0n

### PR DESCRIPTION
**Pull request by DRSchlaubi**
- Fix #439

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,6 +197,19 @@ suspend fun main() {
...
 A mapping of [Discord's Voice Connection](https://discord.com/developers/docs/topics/voice-connections), which maintains the connection and handles audio transmission.
 
+If you want to use voice, you need to enable the voice capability,
+which is only available for Gradle
+
+```kotlin
+dependencies {
+    implementation("dev.kord", "core", "<version>") {
+        capabilities {
+            requireCapability("dev.kord:core-voice:<version>")
```
Comment (HopeBaron) [inline README.md:207]: core-voice?
thought it was just voice

Comment (DRSchlaubi) [reply README.md:207]: This is a variant of core, including voice, the normal voice module remains

**Comment by Lukellmann** [comment]
This already got merged and I don't know anything about gradle feature variants, but why is the config in the [`java {}`](https://github.com/kordlib/kord/pull/608/files#diff-a6fced4b4e4a99784a2b92028b1adbc8c0ca28ff54e264b0e73727b3d4a460cf) block?

**Comment by DRSchlaubi** [comment]
> This already got merged and I don't know anything about gradle feature variants, but why is the config in the [`java {}`](https://github.com/kordlib/kord/pull/608/files#diff-a6fced4b4e4a99784a2b92028b1adbc8c0ca28ff54e264b0e73727b3d4a460cf) block?

It's like the bomb, this kinda tooling is part of gradles java tooling

**Pull request merged**